.github/workflows/test.yml

```diff
@@ -33,8 +33,14 @@ jobs:
         run: |
           forge test --match-path 'test/invariants/*' -vvv
 
-      - name: Run other tests with forking
+      - name: Run other tests with forking Ethereum
         run: |
-          forge test --no-match-path 'test/invariants/*' -vvv
+          forge test --no-match-path 'test/invariants/*' --no-match-contract Radiant -vvv
         env:
           FOUNDRY_ETH_RPC_URL: https://cloudflare-eth.com
+
+      - name: Run other tests with forking Arbitrum
+        run: |
+          forge test --no-match-path 'test/invariants/*' --match-contract Radiant -vvv
+        env:
+          FOUNDRY_ETH_RPC_URL: https://arb1.arbitrum.io/rpc
```

Comment:
Since we have another network, I'm wondering if we need to change `FOUNDRY_ETH_RPC_URL` to `FOUNDRY_RPC_URL` for better naming.

Reply:
I thought that's an environment variable defined by foundry so we cannot change that.

Reply:
Oh right. I thought it was defined in our GitHub. Thanks.